docs/RaidbossGuide.md

```diff
@@ -4,8 +4,13 @@
 
 ## File Structure
 
+Each trigger file is a module that exports a single trigger set.
+
 ```javascript
-[{
+import ZoneId from '../etc/resources/zone_id.js';
```

Comment:
Microscopic nit, but since there's no `/etc/` in the repository structure, it might be more obvious to say something like:
```suggestion
import ZoneId from '../path/to/resources/zone_id.js';
```